national-water-model/xarray-water-model.py

```diff
@@ -0,0 +1,72 @@
+"""
+This example was adapted from https://github.com/dcherian/dask-demo/blob/main/nwm-aws.ipynb
+"""
+
+import coiled
+import dask
+import flox.xarray
+import fsspec
+import numpy as np
+import rioxarray
+import xarray as xr
+
+
+# optionally run with coiled run
+# coiled run --region us-east-1 --vm-type m6g.xlarge python nwm-aws.py
+
+cluster = coiled.Cluster(
+    name="nwm-1979-2020",
+    region="us-east-1", # close to data
+    n_workers=10,
+    scheduler_vm_types="r7g.xlarge", # ARM instance
+    worker_vm_types="r7g.2xlarge",
+    compute_purchase_option="spot_with_fallback" # use spot, replace with on-demand
+)
+
+client = cluster.get_client()
+client.restart()
```

Comment:
Maybe this is meaningful, but my guess is this is left from previous debugging 

```suggestion
```

Reply:
yup, thx!